package.json

```diff
@@ -3,16 +3,12 @@
   "version": "0.0.19",
   "license": "MIT",
   "description": "Regular Expressions parser in JavaScript",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/DmitrySoshnikov/regexp-tree.git"
-  },
+  "repository": "DmitrySoshnikov/regexp-tree",
   "homepage": "https://github.com/DmitrySoshnikov/regexp-tree",
   "bugs": "https://github.com/DmitrySoshnikov/regexp-tree/issues",
-  "main": "index.js",
   "scripts": {
     "build": "./scripts/build",
-    "test": "./scripts/test",
+    "test": "jest",
```

Comment:
Will this work if no global `jest` is installed? The purpose of the script was to use the local version of `jest` installed from `devDependencies`.